Tracking/src/Tracking/Reco/TruthSeedProcessor.cxx

```diff
@@ -610,6 +608,23 @@ void TruthSeedProcessor::produce(framework::Event& event) {
         });
   }
 
+  // Sort tagger hits. TO DO: simplify/merge with above
```

Comment:
I'm not sure if we can merge this with the above. Generally, insertion needs to happen before sorting unless you are willing to change the target data structure (the `std::vector<int>` in this case) to be one that is sorted (like a `std::set`) but since you want the sort to be done by momentum and not index, this could get confusing.